### PR TITLE
[IBM MQ integration package] Update the compatibility section

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR updates the Compatibility section for IBM MQ integration package with the supported official container images.

Closes https://github.com/elastic/integration-docs/issues/559

### CODE REVIEW

File: packages/ibmmq/_dev/build/docs/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Note:
 
 ## Compatibility
 
-This integration has been tested against `IBM MQ v9.1` and `IBM MQ v9.2`. The ibmmq `qmgr` data stream is compatible with a containerized distribution of IBM MQ (since version 9.1.0).
+This integration has been tested against IBM MQ v9.1 and IBM MQ v9.2. Currently, the `ibmmq qmgr` data stream is only compatible with a [containerized distribution of IBM MQ](https://hub.docker.com/r/ibmcom/mq).
```
Comment (tommyers-elastic) [inline packages/ibmmq/_dev/build/docs/README.md:31]: does this suggest we do not support the image from the ibm container registry?

i.e. icr.io/ibm-messaging/mq

**Comment by alaudazzi** [comment]
@tommyers-elastic 
Thank you for your feedback. I'll let @harnish-elastic chime in.

File: packages/ibmmq/_dev/build/docs/README.md (hunk 1 above)
Comment (harnish-elastic) [reply packages/ibmmq/_dev/build/docs/README.md:31]: > does this suggest we do not support the image from the ibm container registry?
> 
> i.e. icr.io/ibm-messaging/mq

I just run the integration for the mentioned image and I am able to collect the metrics. So we do support the image from the ibm container registry. So we can mention that we do support container registry images in the compatibility section.

> i think it's a bit ambiguous, the text says "a containerized distribution", but doesn't specify if it must be a specific containerized distribution (even though the link suggests so). there are many others on docker hub that are not the 'official' image. does it work with these? can we make it more specific?

I have also run the integration for the unofficial images and collected metrics successfully. Let me know your thoughts on the below statement for the compatibility section.

```suggestion
This integration has been tested against IBM MQ v9.1 and IBM MQ v9.2. Currently, the `ibmmq qmgr` data stream is only with the containerized versions of IBM MQ, such as those available from [IBM Cloud Container Registry](https://icr.io/) or [Docker Hub](https://hub.docker.com/r/ibmcom/mq). 
```

Comment (alaudazzi) [reply packages/ibmmq/_dev/build/docs/README.md:31]: Thank you, @harnish-elastic!
I would only complete the sentence with the missing "compatible" word:

> Currently, the `ibmmq qmgr` data stream is only compatible with the containerized versions of IBM MQ

@tommyers-elastic 
Are you happy with these changes? Please let us know if anything else is required.

Comment (harnish-elastic) [reply packages/ibmmq/_dev/build/docs/README.md:31]: > I would only complete the sentence with the missing "compatible" word:

Mb. I have updated the suggestion with missing word. Thanks!

Comment (tommyers-elastic) [reply packages/ibmmq/_dev/build/docs/README.md:31]: 👍 LGTM

**Review by harnish-elastic — APPROVED** [review]
LGTM! Just need to run `elastic-package build` command for CI pass!

**Review by muthu-mps — APPROVED** [review]

**Pull request merged**